### PR TITLE
Improve user experience for Score users

### PR DESCRIPTION
**Pull request by denismakogon**
- multiple improvements for:
	- blueprints operations:
		- new options: --include-plan
		  (includes blueprint plan on demand)
	- deployment operations
	- events operations

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -61,6 +61,6 @@
     zip_safe=True,
     entry_points='''
         [console_scripts]
-        vca=vca_cli.vca_cli:cli
+        vca=vca_cli.vca_cli_impl:cli
```
Comment (kostya13) [inline setup.py]: Why the file name is changed?

**Comment by kostya13** [comment]
Looks good

**Comment by yoramw** [comment]
Hi,

After reviewing the code which looks ok, I want to wait for Paco to review and merge.

**Comment by pacogomez** [comment]
Thanks

**Pull request merged**